cmd/neofs-lens/internal/blobovnicza/inspect.go

```diff
@@ -9,9 +9,17 @@ import (
 )
 
 var inspectCMD = &cobra.Command{
-	Use:   "inspect",
-	Short: "Object inspection",
-	Long:  `Inspect specific object in a blobovnicza.`,
+	Use:        "inspect",
+	Short:      "Object inspection",
+	Long:       `Inspect specific object in a blobovnicza.`,
+	Deprecated: "will be removed in the next release. Use `get` instead.",
+	Run:        inspectFunc,
```

Comment:
let's rename all the `inspectFunc` func to `getFunc` now? you will just drop only `inspectCMD` without any renamings after the release then